integration-tests/baas-test-server/README.md

```diff
@@ -14,7 +14,13 @@ AWS_ACCESS_KEY_ID=...
 AWS_SECRET_ACCESS_KEY=...
 ```
 
-Be sure to have `assisted_agg` (if on Mac) or `libmongo.so` (if on Linux) downloaded and added to your PATH. More info about where to get it can be found [in the BAAS repo](https://github.com/10gen/baas/blob/master/etc/docs/onboarding.md).
+A file called `assisted_agg` needs to be downloaded in order for MongoDB aggregation tests to run:
+1. Download the file `assisted_agg` (if on Mac) or `libmongo.so` (if on Linux) and add it
+   to your `PATH` (see [BaaS Local Environment Setup](https://github.com/10gen/baas/blob/master/etc/docs/onboarding.md)).
+2. Load the `PATH` variable to the terminal window used for starting the BaaS server.
```

Comment:
Maybe better say to remember to update the `PATH` in the profile?

Reply:
Ah, so you mean e.g.: (?)
```md
2. Add the `PATH` to your profile (e.g. to your `~/.profile`, `~/.bash_profile`, or `~/.zshrc`) and make sure it is loaded to the terminal window used for starting the BaaS server.
```

Reply:
Yup!

Reply:
Update: To conform to the [merged changes](https://github.com/realm/realm-js/pull/5483/files) which automatically adds the `assisted_agg` to the `PATH`, are these the correct instructions now @kraenhansen?
```
1. Download the file `assisted_agg` (if on Mac) or `libmongo.so` (if on Linux) (the file is available on [BaaS Local Environment Setup](https://github.com/10gen/baas/blob/master/etc/docs/onboarding.md#third-party-mongo-libraries)).
2. Move the file to the `/integration-tests/baas-test-server/baas` directory.
3. When running the tests, Mac will block execution of the file. Then (for Mac) go
   to `System Settings > Privacy & Security`, find blocked files, then allow `assisted_agg`.
``` 

And does it (should it?) handle `libmongo.so`?